appcues/src/main/java/com/appcues/Appcues.kt

```diff
@@ -167,11 +165,16 @@ public class Appcues internal constructor(koinScope: Scope) {
      * Can be used when the user logs out of your application.
      */
     public fun reset() {
+        // flush any pending analytics for the previous user, prior to reset
+        analyticsTracker.flushPendingActivity()
```

Comment:
moved this responsibility here, from internal in the `sessionMonitor`, as I'm trying to keep the responsibilities separate as much as possible and not rely on the session reset to side effect and flush analytics.